lib/endpoints/class-wp-rest-terms-controller.php

```diff
@@ -140,7 +140,30 @@ public function get_items( $request ) {
 			$response[] = $this->prepare_response_for_collection( $data );
 		}
 
-		return rest_ensure_response( $response );
+		$response = rest_ensure_response( $response );
+		unset( $prepared_args['number'] );
+		unset( $prepared_args['offset'] );
+		$total_terms = wp_count_terms( $this->taxonomy, $prepared_args );
+		$response->header( 'X-WP-Total', (int) $total_terms );
```

Comment:
> I don't know if we ran into this issue with v1.0

We definitely did, WP Engine limits more aggressively than other hosts and we saw problems there. That was only because we sent one-link-per-item though.

Should be fine here (we do the same for posts) for now, I think the plan was to review when we look at the links-for-collections stuff